gapic/ads-templates/%namespace/%name/%version/gapic_metadata.json.j2

```diff
@@ -0,0 +1,33 @@
+{
+    "schema": "1.0",
```

Comment:
If we continue with this template approach, at least specify in both templates that we're adhering to the .proto schema in http://github.com/....